crypto/crypto/Cargo.toml

```diff
@@ -44,6 +45,7 @@ proptest-derive = { version = "0.1.2"}
 ripemd160 = "0.8.0"
 criterion = "0.3"
 sha3 = "0.8.2"
+serde_json = "1.0.51"
```

Comment:
can remove this after deleting noise-cacophony.txt

Reply:
initial thought was to use the file so that people could easily verify that the file is correct. I'm just keeping the relevant portion now but keeping the json struct for the same purpose. Not sure if really useful but it feels like I'm being a good neighbor.